app/models/user.js

```diff
@@ -71,6 +74,38 @@ export default class User extends Model {
     return `${this.firstName} ${this.lastNamePrefix} ${this.lastName}`;
   }
 
+  get fullNickname() {
+    if (this.lastNamePrefix === null) {
+      return `${this.nickname} ${this.lastName}`;
+    }
+
+    return `${this.nickname} ${this.lastNamePrefix} ${this.lastName}`;
+  }
+
+  get fullNameWithNickname() {
+    return this._fullNameWithNickname();
+  }
+
+  get fullNameWithNicknameMd() {
+    return this._fullNameWithNickname(true);
+  }
+
+  _fullNameWithNickname(markdown = false) {
```

Comment:
Nickname on SOFIA
![image](https://user-images.githubusercontent.com/101463776/206002550-19c01f76-c97c-4915-b9c9-240d03fb9e34.png)

Nickname on AMBER
![image](https://user-images.githubusercontent.com/101463776/206002636-11b01daa-b4b1-4449-b72e-aaae563a2294.png)

On AMBER the nickname is put after the lastNamePrefix, but on SOFIA it is put before the lastNamePrefix. I think the one on SOFIA looks a lot better, so can you also put the nickname before the lastNamePrefix here? I believe you just have to sawp two if-statements in this method